### PR TITLE
Add draft themes to SDC sprints

### PR DESCRIPTION
**Pull request by illicitonion**

### CODE REVIEW

File: org-cyf-sdc/content/complexity/sprints/1/_index.md [hunk 1]
```diff
@@ -5,5 +5,5 @@ layout = 'sprint'
 emoji= '⏱️'
 menu_level = ['module']
 weight = 2
-theme = "Playing computer"
+theme = "Memory and CPU consumption/complexity"
```
Comment (SallyMcGrath) [inline org-cyf-sdc/content/complexity/sprints/1/_index.md:8]:
```suggestion
theme = "Memory and cycles"
```

File: org-cyf-sdc/content/computers/sprints/1/_index.md [hunk 2]
```diff
@@ -5,5 +5,5 @@ layout = 'sprint'
 emoji= '⏱️'
 menu_level = ['module']
 weight = 2
-theme = "Playing computer"
+theme = "Shell tools, and binary numbers"
```
Comment (SallyMcGrath) [inline org-cyf-sdc/content/computers/sprints/1/_index.md:8]: No value in 2 themes?

**Review by SallyMcGrath — APPROVED** [review]
Just a question about do you want a list of themes

File: org-cyf-sdc/content/computers/sprints/1/_index.md (hunk 2 above)
Comment (illicitonion) [reply org-cyf-sdc/content/computers/sprints/1/_index.md:8]: To consider for the future :)

**Pull request merged**